VTEX - Marketplace Protocol - External Marketplace Orders.json

```diff
@@ -3515,317 +3208,1004 @@
-                            "type": "object",
-                            "title": "errors",
-                            "required": [
-                                "source",
-                                "code",
-                                "description"
-                            ],
-                            "properties": {
-                                "source": {
-                                    "type": "string",
-                                    "title": "Checkout",
-                                    "description": "Includes the following fields pointing out the context of the error: \n\n`Fulfillment` \n\n`Checkout` \n\n`Order Integration`",
-                                    "example": 
-                                        "Fulfillment"
-                                },
-                                "code": {
-                                    "type": "string",
-                                    "title": "code",
-                                    "description": "String containing the code returned by the source. Example value: If the source is `Fulfillment`, the code can be FMT005 to indicate that the item(s) in the order are not available.",
-                                    "example": "FMT005"
-                                },
-                                "description": {
-                                    "type": "string",
-                                    "title": "description",
-                                    "description": "String containing the error message/description returned by the source.",
-                                    "example": "Message describing the error."
+                    "orderData": {
+                        "type": "object",
+                        "description": "Data related to the created order.",
+                        "properties": {
+                            "marketplaceOrderId": {
+                                "type": "string",
+                                "description": "ID of the order in the marketplace."
+                            },
+                            "marketplaceServicesEndpoint": {
+                                "type": "string",
+                                "description": "Endpoint provided by the marketplace for post purchase communication. Should be an URL, containing protocol, host, path and query string (in case it applies)."
+                            },
+                            "marketplacePaymentValue": {
+                                "type": "integer",
+                                "description": "Value of the payment made to the marketplace."
+                            },
+                            "isCreatedAsync": {
+                                "type": "boolean",
+                                "description": "Indicates whether an order is created. It must be `true` if an order is being placed with Price divergence, otherwise the request will not work."
```

Comment:
```suggestion
                                "description": "Indicates whether an order is created. It must be `true` if an order is being placed with price divergence, otherwise the request will not work."
```